src/faz/client/go_client/faz_client.go

```diff
@@ -0,0 +1,104 @@
+package faz_client
+
+import (
+	"log"
+	"os"
+
+	"github.com/golang/protobuf/ptypes/any"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	faz "github.com/floan/csci499_floan/protos/go_protos/faz"
+)
+
+type FazClient struct {
+	stub_ faz.FazServiceClient
+	conn_ *grpc.ClientConn
+}
+
+// This function acts as a constructor 
+// and initializes the member variables 
+// of the FazClient struct
+func CreateFazClient(address string) *FazClient {
+	// Set up a connection to the server.
+	fazClient := &FazClient{}
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	fazClient.conn_ = conn
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+		os.Exit(1)
+	}
+	// Creating the stub that will call methods
+	fazClient.stub_ = faz.NewFazServiceClient(fazClient.conn_)
+	// Returning our newly created object
+	return fazClient
+}
+
+// This function acts as a destructor 
+// and does what is needed to destroy the 
+// object. NOTE: THIS HAS TO BE CALLED MANUALLY
+func (fazClient *FazClient) DestroyFazClient() {
+	fazClient.conn_.Close()
+}
+
+// This method links to the event method in the server class
+// This executes our stored function via our FaaS Faz
+// Args: A request and response of protobuf Any type. These
+//       represent the input and output from our registered function
+// Returns: A boolean indicating success / failure
+func (fazClient *FazClient) Event(request *any.Any, 
+																	response *any.Any,
+																	eventType int) bool {
+	eventRequest := faz.EventRequest{
+		EventType: int32(eventType), 
+		Payload: request,
+	}
+	eventResponse, err := fazClient.stub_.Event(context.Background(), &eventRequest)
+	if err != nil {
+		log.Fatalf("could not perform event: %v", err)
```

Comment:
I am not sure what is best practice is in Go, but you could have the return type be (error, bool) then return the error message all the way up and log it on the CLI. I think this is fine though.

Reply:
Also I get its close to deadline and you are probably not looking to make major changes

Reply:
I'll definitely look into this, the deadline thing is true so I'll circle back to this after I've finalized the build process!